src/test/engineeffectsdelay_test.cpp

```diff
@@ -86,7 +86,7 @@ TEST_F(EngineEffectsDelayTest, DelayGreaterThanDelayBufferSize) {
     m_effectsDelay.setDelayFrames(numDelayFrames);
 
     const CSAMPLE inputBuffer[] = {-100.0, 100.0, -99.0, 99.0};
-    const CSAMPLE expectedResult[] = {-100.0, 75.0, -49.5, 24.75};
+    const CSAMPLE expectedResult[] = {-50.0, 50.0, -49.5, 49.5};
 
     mixxx::SampleBuffer pInOut(numSamples);
```

Comment:
I just noticed another flaw in these tests (sorry for widening the scope of the PR). 
the `p` prefix of variables is only used for variables holding a pointer. But samplebuffer is not a pointer, so we should fix the name accordingly.

Reply:
No problem. You're right, it does not make sense.